api/internal/handler/service/service.go

```diff
@@ -89,6 +89,42 @@ type ListInput struct {
 	store.Pagination
 }
 
+// swagger:operation GET /apisix/admin/services getServiceList
+//
+// Returns service list
+//
+// Return the service list according to the specified page number and page size, and can search by name
```

Comment:
English Style: keep the singular consistent.

Reply:
fixed.